MAUI/DataForm/grouping.md

```diff
@@ -0,0 +1,386 @@
+---
+layout: post
+title: Grouping the editors in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the grouping support in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Grouping the editors in .NET MAUI DataForm (SfDataForm)
+
+The .NET MAUI DataForm support to group the editors which are relevant to each other. You can expand or collapse the group by tapping the group item.
+
+Grouping can be achieved by defining [Display](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute?view=net-7.0) attribute or by handling the `GenerateDataFormItem` event.
+
+#### Using attributes
+
+{% tabs %}
+{% highlight C# %}
+
+public class ContactsInfo
+{
+    [Display(GroupName = "Name")]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Name")]
+    public string MiddleName { get; set; }
+
+    [Display(GroupName = "Name")]
+    public string LastName { get; set; }    
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName" || e.DataFormItem.FieldName == "MiddleName" || e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.GroupName = "Name";
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing editors order in the group
+
+#### Using attribute
+
+#### Display attribute
+
+The order of the editors in the group can be changed by using attributes. You can set the order of data form items in group by using the `Order` property along with `GroupName` property in `Display` attribute.
+
+{% tabs %}
+{% highlight C# %}
+public class ContactInfo
+{
+    [Display(GroupName = "Name", Order = 0)]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Name", Order = 2)]
+    public string LastName { get; set; }
+
+    [Display(GroupName = "Name", Order = 1)]
+    public string MiddleName { get; set; }
+}
+{% endhighlight %}
+{% endtabs %}
+
+#### DataForm display options attribute
+
+The order of the editors in the group can also be changed by using `RowOrder` property of [DataFormDisplayOptions] attribute.
```

Comment:
The order of the editors in the group is also changed by using the `RowOrder` property of the [DataFormDisplayOptions] attribute.

Reply:
Content changed